<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureReportDialog</class>
 <widget class="QDialog" name="ConfigureReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Alarm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="17,20">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Triggers</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="16" column="0">
          <widget class="QLabel" name="label_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>FTP Password</string>
           </property>
          </widget>
         </item>
         <item row="16" column="1">
          <widget class="QLineEdit" name="ftpPassword">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>FTP Username</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="useCustomPeriod">
           <property name="text">
            <string>Custom Period</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Period</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="ftpServer">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sends an email report to this address</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name">
           <property name="text">
            <string>Report</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QCheckBox" name="uploadToFtpAction">
           <property name="text">
            <string>Upload To FTP Server</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/ftp.png</normaloff>:/icons/ui/ftp.png</iconset>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="sendEmailAction">
           <property name="text">
            <string>Send Email To</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/email.png</normaloff>:/icons/ui/email.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="period">
           <item>
            <property name="text">
             <string>Daily</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Weekly</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Monthly</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="customPeriod">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> day</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="ftpFolder">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter a unique name for this report</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Uploads the entire DB and a html report to this FTP server</string>
           </property>
          </widget>
         </item>
         <item row="18" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>FTP Folder</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="ftpUsername">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="emailRecipient">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>How often to send the report</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QRadioButton" name="summaryData">
           <property name="text">
            <string>Summary Data</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QRadioButton" name="allData">
           <property name="text">
            <string>All Data</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sensorGroup">
        <property name="title">
         <string>Sensors</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTreeView" name="sensorList">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureReportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureReportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendEmailAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>emailRecipient</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>period</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomPeriod</sender>
   <signal>toggled(bool)</signal>
   <receiver>customPeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>ftpServer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>ftpUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>ftpPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>44</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadToFtpAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>ftpFolder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
